test/test_mps.py

```diff
@@ -3732,6 +3732,13 @@ def helper(start, end, steps, dtype=torch.float32):
             helper(5, 2, 10, dtype)
             helper(2, 2, 0, dtype)
 
+    # Test argange
+    def test_arange(self):
+        self.assertEqual(torch.tensor(np.arange(10)), torch.arange(10, device='mps'))
```

Comment:
nit: the tensor is not needed here are default integer dtypes match for np and pytorch.
For the floating point versions below, passing `dtype=np.float32` to the numpy constructor works.